apps/condo/domains/notification/access/Device.js

```diff
@@ -0,0 +1,40 @@
+// @ts-nocheck
+/**
+ * Generated by `createschema notification.Device 'deviceId:Text; pushToken?:Text; pushTransport?:Select:firebase,apple,huawei; owner?:Relationship:User:SET_NULL; meta?:Json'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+/**
+ * Manages native readability of schema. Is readable by admin and and authorized owner user.
+ * @param user
+ * @returns {Promise<{}|boolean|{owner: {id}}>}
+ */
+async function canReadDevices ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
```

Comment:
What about blocked user?
```suggestion
    if (user.isBlocked) return false
    if (user.deletedAt) return false
```

Reply:
fixed

Reply:
We don’t have isBlocked

Reply:
Ah, correct. `isBlocked` is presented for employee